pkg/cmd/server/origin/master_config.go

```diff
@@ -434,6 +434,11 @@ func (c *MasterConfig) ImageImportControllerClient() *osclient.Client {
 	return c.PrivilegedLoopbackOpenShiftClient
 }
 
+// DeploymentConfigScaleClient returns the client used by the Scale subresource registry
+func (c *MasterConfig) DeploymentConfigScaleClient() *kclient.Client {
```

Comment:
See the stub here: https://github.com/openshift/origin/pull/5357 for getting a properly scoped client.